### ISSUE TITLE
Display glitches on HiDPI screens

### ISSUE THREAD

**Issue by mokrinsky** [simulationcraft/simc#3929]
I'm sure there are not so many users with HiDPI or 4K screens, but still... There is a bug in displaying SimulationCraft on such screens. Actually it looks like [here](https://user-images.githubusercontent.com/463907/28543232-c169479e-70c7-11e7-99bf-d4e6798a4ad9.png). And also I have 15" display, so you could imagine how small it is. I'm not sure what to add, it's kinda not a bug but just a thing to implement, but if you need some additional information - I'll provide.

**Comment by Strit** [comment]
The window it self seems to be alright, but the contents displayed is not, as it is simple html output.
Anything in HTML that can scale with HiDPI?

**Comment by navv1234** [comment]
A much better designed HTML report.

**Comment by mokrinsky** [comment]
Not really. HTML is not a real problem cause every report is saved in simc folder so I'm able ot open it in browser. But the annoying part is tabs that are hidden sometimes (like on screenshot - there are almost no tabs). However, clickable areas are working, so if I click on empty space where tab should be, It will be clicked and tab content will be shown, however still no tabs displayed.

**Comment by nullKomplex** [comment]
I'm not even running HiDPI or 4k and I experience a very similar bug. Running 1920x1080. Makes it a little frustrating to use. The weirdest part is (as this is a fairly new laptop) that I ran one nightly build just fine, and then about a week or so later ran the exact same build and experienced the bug, and so far it hasn't gone away (even a restart). I did make some changes since then (that I cannot remember) so I wonder if anything was related.

nVidia 1060. Forcing it to run on the GPU makes it completely unusable as it only renders the initial frame upon opening a tab and nothing after (swapping to an HTML tab and back to that one will update it).

EDIT: Figured out my first problem (the one most similar to OP). I removed Acer's intel HD drivers and installed the regular ones and it works fine when using Intel HD as the graphics processor. Using the 1060 GPU still has those weird problems but I don't expect to run it on that anyway.
